packages/components/organisms/f-header/src/components/Navigation.vue

```diff
@@ -331,6 +344,12 @@ export default {
         showCountrySelector: {
             type: Boolean,
             default: false
+        },
+
+        customNavLinks: {
+            type: Array,
+            default: () => [],
+            validator: links => links.every(link => link.text && link.url)
```

Comment:
Would an Object containing GTM data produce a validation warning here?

Reply:
@zlatinivanov-je The `gtm` object isn't currently required, so I haven't included it in the validation, what kind of validation did you have in mind?

Reply:
Looking at the code and trying it my self did not come with a good solution for what I was proposing. 
Resolving the conversation. 
Just wanted to try and flag if something is being missed.